src/api_server/src/lib.rs

```diff
@@ -524,7 +525,7 @@ mod tests {
         let to_vmm_fd = EventFd::new(libc::EFD_NONBLOCK).unwrap();
         let (api_request_sender, _from_api) = channel();
         let (_to_api, vmm_response_receiver) = channel();
-        let mmds_info = Arc::new(Mutex::new(Mmds::default()));
+        let mmds_info = Arc::new(Mutex::new(Mmds::new().unwrap()));
```

Comment:
Add a comment stating why the unwrap is safe

Reply:
At a second 👀  this is not safe, as `Mmds::new` will return an error if TokenAuthority fails

Reply:
This unwrap is in a test module I was talking about this one https://github.com/firecracker-microvm/firecracker/blob/32c12c82472189330d4f61d7b62c70feecd63b5f/src/mmds/src/lib.rs#L25. But I think expect is OK here so will resolve this